tests/everything.values.yaml

```diff
@@ -142,6 +142,7 @@ ingress:
     enabled: false
     loadBalancerType: ROUND_ROBIN
     mtlsMode: "PERMISSIVE"
+    forceHttpRedirect: false
```

Comment:
I think you may have added a lot of new stuff, so test case needs to be updated I think to take into account hte new stuff.

Reply:
While we're at it we might refactor tests to use https://helm.sh/docs/topics/chart_tests/